assets/xml/code/eff_footmark.xml

```diff
@@ -0,0 +1,9 @@
+<Root>
+    <File Name="code" BaseAddress="0x800A5AC0" RangeStart="0x116750" RangeEnd="0x1167E0" OutName="eff_footmark">
+        <Array Name="gEffFootprintsVtx" Count="3" Offset="0x116750">
+            <Vtx/>
+        </Array>
+        <DList Name="gEffFootprintsMaterialDL" Offset="0x116780"/>
+        <DList Name="gEffFootprintsModelDL" Offset="0x1167C8"/>
```

Comment:
does it draw a single footprint or multiple?
singular sounds better to me here generally speaking, but if it draws 2 for example then maybe its fine

Reply:
Ohh nice catch. It only draws one